lxd/main_import.go

```diff
@@ -1,28 +1,45 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
 	"net/http"
 
 	"github.com/lxc/lxd"
+	"github.com/lxc/lxd/shared"
 	"github.com/lxc/lxd/shared/api"
+	"github.com/lxc/lxd/shared/version"
 )
 
 func cmdImport(args []string) error {
 	name := args[1]
+	b := shared.Jmap{
```

Comment:
@stgraber, what do you think? Afaik, POST should always sent any data in the body and not as part of the query string. I thought adding another `?target=%s&force=%s` would be even weirder and so switched this over. I think it should be fine since we don't expect users to use two different LXD binaries between daemon and the import command and also this is internal api.